.github/workflows/telemetry-client-ci.yml

```diff
@@ -0,0 +1,44 @@
+
+name: 'Telemetry Client CI'
+on:
+    push:
+      paths:
+       - 'projects/user-telemetry-client/**'
+    pull_request:
+      paths:
+       - 'projects/user-telemetry-client/**'
+
+jobs:
+    ci:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout branch
+            uses: actions/checkout@v2
+          - uses: actions/setup-node@v2
+            with:
+              node-version-file: '.nvmrc'
+          - run: npm install
+            working-directory: ./projects/user-telemetry-client
+          - run: npm test
+            working-directory: ./projects/user-telemetry-client
+
+    release:
```

Comment:
IIUC because we don't have to orchestrate other workflows for this client, we could just include this as a separate, conditional step, which would save us repeating our checkout-install-build process. (The reason we repeat these steps in prosemirror-elements is to parallelise our integration test and build/test steps). This is non-blocking, it doesn't matter too much.

Reply:
cf. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif

Reply:
Good spot, I've amended!